contentPages/models.py

```diff
@@ -10,6 +10,7 @@
 from wagtail.snippets.edit_handlers import SnippetChooserPanel
 from wagtail.snippets.models import register_snippet
 from CMS.enums import enums
+from django.contrib.contenttypes.models import ContentType
```

Comment:
Looks like this import is added but never used

Reply:
Deleted!